### DISCUSSION TITLE
Adding block nodes to a new Database entry

### DISCUSSION THREAD

**Discussion by krazzarkov** [ramnes/notion-sdk-py#173] [Q&A]
Hello,

I have been trying to figure out this issue for the past couple of days and am completely stumped, was hoping someone who understands the Notion API better can offer some tips/help.

I currently have a Python script that does the following:
1. Goes through a folder of CSVs
2. Reads data/rows of CSVs and uploads the data collected from the CSV files as a new page inside my Notion database.
3. For certain data from my CSV, I need to upload them as blocks inside the actual page that I created in step 2. 

Step 3 is what I am having issues with. So far my script is able to create a new page inside the database with various properties. However, I cannot figure out a way to add text blocks inside the newly created page. 

I have looked through the documentation but have not been able to find a solution. My guess is I need to get the ID of the page that I created and then add a block for that specific page ID. 

Any help would be greatly appreciated. Thank you in advance :)

**Comment by faizel-noorgat** [comment]
Have you found a solution for this
